docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -245,6 +245,18 @@ Total size of all shards assigned to the node.
 `size_in_bytes`::
 (integer)
 Total size, in bytes, of all shards assigned to the node.
+
+`reserved`::
+(<<byte-units,byte value>>)
+A prediction of how much larger the shard stores on this node will eventually
+grow due to ongoing peer recoveries, restoring snapshots, and similar
+activities. A value of `-1b` indicates that this is not available.
```

Comment:
`GET _nodes/stats?level=shards` reports this value on a shard-by-shard basis, and includes initialising shards, so it could well be unknown in some of these cases.

Reply:
I think that on a shard by shard basis, -1 could make sense (though I find it hard to think of a case where it matters).

But on a summarized level, getting -1 out because the stats are pulled at an unfortunate time seems to hide more information than it gives.